src/components/asset-list/RecyclerAssetList2/FastComponents/FastUniswapPoolRow.tsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import { StyleSheet, View } from 'react-native';
+import { ButtonPressAnimation } from '../../../animations';
+import FastCoinIcon from './FastCoinIcon';
+import FastPoolValue from './FastPoolValue';
+import { Text } from '@rainbow-me/design-system';
+import { supportedNativeCurrencies } from '@rainbow-me/references';
+
+interface UniswapCoinRowItem {
+  onPress: () => void;
+  tokens: any[];
```

Comment:
I think we might have somewhere a type for this

Reply:
CC @marco Maybe you know something that can work here?

Reply:
@tchayen Correct, there are a few token types located in [`src/entities/tokens`](https://github.com/rainbow-me/rainbow/blob/aeb8cdfca8eef3ca4b559e46d16fa0b5f2647fb7/src/entities/tokens.ts), however from looking at the code I think the type that actually fits best might be [`UniswapPoolAddressDetailsFull`](https://github.com/rainbow-me/rainbow/blob/b8b4f082473eb7086c312c50045e955853546c48/src/redux/uniswapLiquidity.ts#L35), since the data is ultimately loaded from the `useUniswapLiquidity` hook. Currently that type is not exported, (it is in a future PR but that has not been merged yet), so you might want to export it from that file and then use it?